README.md

```
@@ -380,6 +382,123 @@ Add the `lazy` attribute to defer a component's loading until it scrolls into vi
+
+  async function send() {
+    const res = await apiFetch('/api/send', { method: 'POST' })
+    if (res.ok) toast('Sent!')
+  }
+</script>
+```
+
+`inject` returns `undefined` (or the provided fallback) during prerender so components remain SSG-safe.
+
+> **Context scope:** `provide` / `inject` share a single app-level context map that lives for the whole page session — values persist across SPA navigations and are not scoped per route. Use `Yon.provide` in `main.js` for app-wide services. Calling `provide` from a component script is valid but the value will remain available globally for the rest of the session.
+
```

markdownlint-cli2 (0.22.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)